src/components/HeatMap/Row/Row.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Cell from '../Cell';
+import { Wrapper, RowName, Text } from './Row.styles';
+
+
+export default function Row({
+  data, dayLabel, row, getActiveCell, activeCell,
+}) {
+  return (
+    <Wrapper>
+      <RowName>
+        <Text>
+          {dayLabel}
+        </Text>
+      </RowName>
+      {
+        data.map((item, i) => (
+          <Cell
+            key={`${row}${i * 2}`}
```

Comment:
Why do you calculate the key like this? Since this rendered array is nested inside a wrapped component you can simply use the index. The key doesn't need to be unique globally

Reply:
I couldn't get rid of the slint error even with an explicit disable next line :(

Reply:
This works for me

```
<Cell
  // eslint-disable-next-line react/no-array-index-key
 key={i}
```

Reply:
Yes, now it worked also. Maybe I had a typo